tests/optim/helpers/numpy_transforms.py

```diff
@@ -53,36 +55,74 @@ def jitter(self, x: np.ndarray) -> np.ndarray:
         return self.translate_array(x, insets)
 
 
-class CenterCrop(object):
+class CenterCrop:
     """
-    NumPy version of the CenterCrop transform
+    Center crop a specified amount from a tensor.
+    Arguments:
+        size (int, sequence, int): Number of pixels to center crop away.
+        pixels_from_edges (bool, optional): Whether to treat crop size values
+            as the number of pixels from the tensor's edge, or an exact shape
+            in the center.
     """
 
-    def __init__(self, size=0) -> None:
-        super().__init__()
-        if type(size) is list or type(size) is tuple:
-            assert len(size) == 2, (
-                "CenterCrop requires a single crop value or a tuple of (height,width)"
-                + "in pixels for cropping."
-            )
-            self.crop_val = size
-        else:
-            self.crop_val = [size] * 2
-        assert len(self.crop_val) == 2
-
-    def crop(self, input: np.ndarray) -> np.ndarray:
-        assert input.ndim == 3 or input.ndim == 4
-        if input.ndim == 4:
-            h, w = input.shape[2], input.shape[3]
-        elif input.ndim == 3:
-            h, w = input.shape[1], input.shape[2]
-        h_crop = h - self.crop_val[0]
-        w_crop = w - self.crop_val[1]
+    def __init__(
+        self, size: IntSeqOrIntType = 0, pixels_from_edges: bool = False
+    ) -> None:
+        super(CenterCrop, self).__init__()
+        self.crop_vals = size
+        self.pixels_from_edges = pixels_from_edges
+
+    def forward(self, input: np.ndarray) -> np.ndarray:
+        """
+        Center crop an input.
+        Arguments:
+            input (array): Input to center crop.
+        Returns:
+            tensor (array): A center cropped tensor.
+        """
+
+        return center_crop(input, self.crop_vals, self.pixels_from_edges)
+
+
+def center_crop(
+    input: np.ndarray, crop_vals: IntSeqOrIntType, pixels_from_edges: bool = False
+) -> np.ndarray:
+    """
+    Center crop a specified amount from a array.
+    Arguments:
+        input (array):  A CHW or NCHW image array to center crop.
+        size (int, sequence, int): Number of pixels to center crop away.

```

Comment:
do we need size here ? Shouldn't it be crop_vals ?

Reply:
Yeah, oops!